resume.md

```diff
@@ -1,49 +1,43 @@
-# FIRST LAST NAME
+# MAANAV KHAITAN
 
-(123) 456 - 7890 | email@gmail.com
+(510) XXX - XXXX | maanav@berkeley.edu
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+github.com/MaanavKhaitan | linkedin.com/in/maanav-khaitan
```

Comment:
Definitely consider making a website of your own - even static websites are great to flex your work